support/reftest/reftest

```diff
@@ -255,15 +520,15 @@ def download_to_local(url, key_path, cert_path, cacert_path):
         return temp_file, sha256.hexdigest()
 
 
-def prepare(db_client, s3_client, config, opt):
+def prepare(publisher: PublishBackend, config, opt):
+    publisher.start_publish()
     for item in config.test_data:
         if not item.get("deploy", True):
             # skip items that specify deploy=false
             continue
 
         if item.get("state") == "absent":
-            # skip all the downloading and checking when absent=true
-            db_client.put_absent_item(item["path"])
+            # Just don't put any item.
             continue
```

Comment:
IIRC, we were putting an "absent" item on the db to test that it wasn't retrievable. Will that functionality remain covered?

Reply:
I don't think this particular scenario:

- If an item exists in DB with state 'absent', trying to fetch it gives a 404

...is covered by reftest data now.  I don't think it's a problem though.  It was never my intent for reftest data to cover that.  It's basically just supposed to declare the expected state of the CDN, not declare what should be deployed in test environments.

The only absent item is:

```
- path: /content/dist/rhel8/8.2/x86_64/baseos/iso/does-not-exist.iso
  state: absent
```

...which was added by me, and when adding it I definitely had in mind that I'm trying to declare "this is a URL which we know doesn't contain any content" (therefore fetching it should give 404).  And not "this is a URL which should have an 'absent' item explicitly provisioned in each environment" (therefore fetching it should give 404, but for a different reason).